src/sidebar/components/TagEditor.tsx

```diff
@@ -262,12 +262,6 @@ function TagEditor({
         id={tagEditorId}
         data-testid="combobox-container"
         ref={closeWrapperRef}
-        // Disabled because aria-controls must be attached to the <input> field
-        // eslint-disable-next-line jsx-a11y/role-has-required-aria-props
-        role="combobox"
-        aria-expanded={suggestionsListOpen.toString()}
-        aria-owns={`${tagEditorId}-AutocompleteList`}
```

Comment:
Given the DOM structure of this component, this attribute did not seem necessary.